nservicebus/upgrades/5to6.md

```diff
@@ -632,7 +633,7 @@ snippet: 5to6-MsmqSubscriptionAuthorizer
 
 ## Synchronous request-response (callbacks)
 
-The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6. 
+The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6.
```

Comment:
is there a reason for the trailing spaces or why to keep them?

Is this configurable in Atom?